libs/datamodel/core/src/configuration/source/builtin/mssql_source.rs

```diff
@@ -0,0 +1,39 @@
+use crate::configuration::*;
+use datamodel_connector::{Connector, ExampleConnector};
+
+pub const MSSQL_SOURCE_NAME: &str = "sqlserver";
```

Comment:
pure bikeshedding: any reason to go for `"sqlserver"` rather than `"mssql"`?

Reply:
The standard indicator for SQL Server is `sqlserver`, like a JDBC URL for SQL Server starts with `sqlserver://` or `jdbc+sqlserver://`, so I guess it's ok to use that in here too...